### ISSUE TITLE
tests fail with libgeotiff-1.7.1

### ISSUE THREAD

**Issue by t0b3** [PDAL/PDAL#3733]
**Describe the bug**
ci tests fail with libgeotiff-1.7.1
- pdal_spatial_reference_test see https://github.com/PDAL/PDAL/runs/5779533515?check_suite_focus=true#step:7:1137
- pdal_filters_reprojection_test see https://github.com/PDAL/PDAL/runs/5779533515?check_suite_focus=true#step:7:3059

bisecting showed this occurs due to https://github.com/OSGeo/libgeotiff/commit/aa98b78732179dc3812e24c90b6f275a57f24ad7

**Expected behavior**
passing tests

**Comment by abellgithub** [comment]
This looks like some file with bad geotiff data not previously detected.

**Comment by hobu** [comment]
Yes. It was common a long time ago for the LAS community to write crappy GeoTIFF keys. I assume this is a really old test file in our suite that's failing, but we should more gracefully fail here.
